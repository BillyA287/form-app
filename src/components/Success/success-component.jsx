import React from "react";
import Dialog from "@material-ui/core/Dialog";
import AppBar from "@material-ui/core/AppBar";

export class Success extends React.Component {
  continue = (event) => {
    event.preventDefault();
    // send form
    this.props.nextStep();
  };

  back = (event) => {
    event.preventDefault();
    this.props.previousStep();
  };
  render() {
    return (
     
        <>
          <Dialog open fullWidth maxWidth="sm">
            <AppBar title="Success" />
            <h1>Thank You For Your Submission</h1>
            <p>You will get an email with further instructions.</p>
          </Dialog>
        </>
      
    );
  }
}



export default Success;
