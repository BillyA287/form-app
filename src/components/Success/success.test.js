import React from 'react'
import Success from './success-component'
import {shallow} from 'enzyme'


it('correctly renders Success component', ()=> {
    expect(
      shallow(<Success continue={jest.fn()} back={jest.fn()} />)
    ).toMatchSnapshot();
})

