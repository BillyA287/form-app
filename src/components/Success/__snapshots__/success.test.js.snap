// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders Success component 1`] = `ShallowWrapper {}`;
